lib/y2lan_restart_common.pm

```diff
@@ -38,6 +43,13 @@ our @EXPORT = qw(
 );
 my $module_name;
 
+=head2 initialize_y2lan
+
+ initialize_y2lan();
+
+Initialize yast2 lan. Stop firewalld, enable debug for detailed messages and easier detection of restart, clear journal.log 
```

Comment:
You could shorten the line rephrasing these items:
`s/Stop firewalld/Ensure firewalld is stopped/`
`s/enable debug for detailed messages and easier detection of restart/Enable DEBUG/`
`s/clear journal.log/Clear the journal/`